addons/controllers/vspherecpiconfig_controller_test.go

```diff
@@ -201,6 +209,11 @@ var _ = Describe("VSphereCPIConfig Reconciler", func() {
 				Expect(strings.Contains(secretData, "supervisorMasterEndpointIP: supervisor.default.svc")).Should(BeTrue())
 				Expect(strings.Contains(secretData, "supervisorMasterPort: \"6443\"")).Should(BeTrue())
 
+				// assert that non paravirt data values don't exist, the keys should not exist
+				Expect(strings.Contains(secretData, "datacenter:")).Should(BeFalse())
```

Comment:
this will throw the following error

```
test-val.yaml:
     |
   9 |     datacenter:
     |

     = found: null
     = expected: string (by schema.yaml:15)
     = hint: is the default value set using @schema/default?
```

since we haven't add nullable here https://github.com/vmware-tanzu/community-edition/blob/72073e438757cddd12b91f2f1673419dcb388d5b/addons/packages/vsphere-cpi/1.23.0/bundle/config/schema.yaml#L14

Reply:
This is resolved with https://github.com/vmware-tanzu/community-edition/pull/4135